### PR TITLE
feat(Unstable_SvgIcon): initial implementation

### PR DESCRIPTION
**Pull request by WilliamKelley**
Implements PDS v2 SvgIcon. Part of https://github.com/prenda-school/design-system/issues/350.

Note that this component is not spec'd by the "components" Figma file (to my displeasure). Font sizes are extrapolated from ["icons" file, "Cover" page](https://www.figma.com/file/BNkemx3eS6ayFVn4mij6tO/?node-id=410%3A50). Colors are extrapolated from color palette comments (see screenshot below); `"normal"` was used instead of `"default"` since the default is actually `"inherit"`, so that would have been a misnomer. As in the previous version of the component. `"inherit"` is the default for font size and color because it makes styling in components **much** easier. Also, icons are used predominantly in, not outside of, components that consider them in styling.

image

### CODE REVIEW

File: libs/spark/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,40 @@
...
+- **Unstable_SvgIcon**
+  - Initial implementation of `SvgIcon` replacement according to PDS v2.
+  - Supports rendering _without_ `theme` being in an ancestor `ThemeProvider`.
+  - CSS API Changes:
+    - Removed all class keys _except `"root"`_.
+  - Props API Changes:
+    - `classes`
+      - Removed all class keys _except `"root"`_.
+    - `color`
+      - Removed values: `"dark"`, `"light"`, `"white"`, `"disabled"`, `"error"`, `"success"`, `"warning"`, `"info"`.
```
Comment (shadoath) [inline libs/spark/CHANGELOG.md:18]: With the error/success/warning/info removed, is there a way to use these in notices that have these settings?

File: libs/spark/src/Unstable_SvgIcon/Unstable_SvgIcon.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  </Unstable_SvgIcon>
+);
+
+export const Playground: Story = Template.bind({});
+
+export const Default: Story = Template.bind({});
```
Comment (shadoath) [inline libs/spark/src/Unstable_SvgIcon/Unstable_SvgIcon.stories.tsx:26]: What does `Template.bind({})` do? what might be passed as params?

File: libs/spark/CHANGELOG.md (hunk 1 above)
Comment (shadoath) [reply libs/spark/CHANGELOG.md:18]: or do we now have to be explicit?

Comment (shadoath) [reply libs/spark/CHANGELOG.md:18]: Or from what we did today it will inherit what color it needs to be, right?

**Review by shadoath — APPROVED** [review]
What

File: libs/spark/src/Unstable_SvgIcon/Unstable_SvgIcon.stories.tsx (hunk 2 above)
Comment (WilliamKelley) [reply libs/spark/src/Unstable_SvgIcon/Unstable_SvgIcon.stories.tsx:26]: It lets you reuse the template, but have the controls panels be separate for every story, so they don't all affect each other.

**Comment by WilliamKelley** [comment]
> What

😅 yes?

**Review by slingner — APPROVED** [review]

File: libs/spark/src/Unstable_SvgIcon/Unstable_SvgIcon.stories.tsx (hunk 2 above)
Comment (WilliamKelley) [reply libs/spark/src/Unstable_SvgIcon/Unstable_SvgIcon.stories.tsx:26]: Also I dont think any params can be passed. it's like [`Function.prototype.bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) I think

Comment (WilliamKelley) [reply libs/spark/src/Unstable_SvgIcon/Unstable_SvgIcon.stories.tsx:26]: resolving for merge. feel free to elicit more discussion though

**Pull request merged**